src/Routers/AppRouter.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import CardsContainer from '../Components/Cards/CardsContainer'
+import Login from '../Components/Login/Login'
+
+const AppRouter = () => {
```

Comment:
If you wish to keep this component just remember to place it as a child of the MainLayout or at least inside the AuthProvider in case you plan to use different layouts. And place the Navigation component inside the Router for the links to work 😉